### PR TITLE
Set up sentry hook

### PR DESCRIPTION
**Pull request by conbrad**
- Doesn't add feedback form just captures errors and replays
- Enabled for backend as well
# Test Links:
[Landing Page](https://wps-pr-3569.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-3569.apps.silver.devops.gov.bc.ca/morecast)
[Percentile Calculator](https://wps-pr-3569.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-3569.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-3569.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3569.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3569.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3569.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3569.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/src/components/FeedbackButton.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { Button } from '@mui/material'
+import CampaignIcon from '@mui/icons-material/Campaign'
+import { feedbackIntegration } from '@sentry/react'
+
+const FeedbackButton = ({ color }: { color: 'primary' | 'inherit' }) => {
```
Comment (conbrad) [inline web/src/components/FeedbackButton.tsx:6]: Not used anywhere yet, just ready to be picked up

**Review by dgboss — APPROVED** [review]
Looks good! Just a couple questions/nits:

1. Are the values for sentry_dsn for the front end and back end supposed to be different?  The secret for the API is different than the value in the configmap for the front-end.
2. Does the sentry_dsn for the API need to be in a secret (or would a config map do)? I'm just curious, it doesn't matter to me!
3. Maybe consider updating the API's .env.example file as a reminder of the new env variable.
Nice work!

**Comment by conbrad** [comment]
> 1. Are the values for sentry_dsn for the front end and back end supposed to be different?  The secret for the API is different than the value in the configmap for the front-end.

Yep, we configure a project for the frontend and backend separately and it looks like sentry seperates dsn by project

> 2. Does the sentry_dsn for the API need to be in a secret (or would a config map do)? I'm just curious, it doesn't matter to me!

I think a secret would be best, otherwise if leaked we could get stack traces from other codebases

> 3. Maybe consider updating the API's .env.example file as a reminder of the new env variable.

Will do, good call!

**Comment by dgboss** [comment]
Cool, thanks for the info!

**Pull request merged**